datastore/blob/azure_storage_api/__init__.py

```diff
@@ -112,21 +112,22 @@ async def get_blob(container_client, blob_name):
         raise GetBlobError("Error getting blob")
 
 
-async def upload_image(container_client, folder_uuid, image:str, image_uuid):
+async def upload_image(container_client, folder_name, folder_uuid, image:str, image_uuid):
     """
     uploads the image to the specified folder within the user's container,
     if the specified folder doesnt exist, it creates it with a uuid
     
     Parameters:
     - container_client: the Azure container client
-    - folder_uuid: the name of the destination folder
+    - folder_name: the name of the destination folder
+    - folder_uuid : uuid of the picture_set
     - image:
     """
     try:
-        if not await is_a_folder(container_client, folder_uuid):
-            raise CreateDirectoryError(f"Folder:{folder_uuid} does not exist")
+        if not await is_a_folder(container_client, folder_name):
+            raise CreateDirectoryError(f"Folder:{folder_name} does not exist")
         else:
-            blob_name = "{}/{}.png".format(folder_uuid, image_uuid)
+            blob_name = "{}/{}.png".format(folder_name, image_uuid)
```

Comment:
How does this work with the different image formats? jpg, tif, tiff . do we save them all as png?

Reply:
Yes, I get something like this from the front : `"data:image/tiff;base..,.........."` which is then hashed by the backend and then it's saved in blob storage with this blob_name so with a png format.

Reply:
yeah that's confusing. We should use .blob or .hash or omit the extension if not relevant.